### ISSUE TITLE
Luminosity contrast ratio of '--rz-secondary' text with its background does not meet the minimum contrast ratio of 4.5:1 in Windows.

### ISSUE THREAD

**Issue by msftedad** [radzenhq/radzen-blazor#1923]
**Describe the bug**
Luminosity contrast ratio of '--rz-secondary' text with its background does not meet the minimum contrast ratio of 4.5:1 in Windows. The low contrast ratio between the '--rz-secondary' text and its background in Windows can greatly impact users with visual impairments. This issue makes it challenging for them to read the text, which can result in missed information and a diminished user experience.

**To Reproduce**

1. Open URL: [Blazor Color Utilities | Free UI Components by Radzen](https://blazor.radzen.com/colors?theme=material) in the latest edge browser.
2. Navigate to '--rz-secondary' content in theme colors section.
3. Turn on the accessibility insights for windows tool.
4. Select Foreground color (#FFFFFF) and Background color (#E91E63) of 'Search' and 'more options' icon using color picker.
5. Verify whether contrast ratio between '--rz-secondary' text with its background does meet the minimum contrast ratio or not.

Alternatively link your repo with a sample project that can be run.
https://blazor.radzen.com/colors?theme=material

**Expected behavior**
Contrast ratio between '--rz-secondary' text with its background should be greater than or equal to the required ratio of 4.5:1 in Windows.

**Screenshots**

https://github.com/user-attachments/assets/88fe1551-d194-4a0c-8fe2-7af3086b74cc

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/59c9d014-b17a-48c1-a585-3eab9a6355fd" />

**Desktop (please complete the following information):**
OS: Windows 11 Enterprise Insider Preview Version 24H2(OS Build 26120.2415)
Browser: Microsoft Edge Version 132.0.2957.26

**Additional context**
NA

**Comment by msftedad** [comment]
Hi @yordanov, Issue is still repro'ing at our end. Could you please reactivate this issue?

**Comment by yordanov** [comment]
Hi @msftedad,

The contrast ratio between --rz-secondary color and background color is 4.54:1 covering the WCAG AA requirements.

<img width="729" alt="Image" src="https://github.com/user-attachments/assets/420c3b9e-d0cb-4c73-9508-c8b043e5376d" />

Please share if there’s anything else you think should be modified.
